lvgl/Cargo.toml

```diff
@@ -77,7 +77,7 @@ unsafe_no_autoinit = []
 quote = "1.0.23"
 proc-macro2 = "1.0.51"
 lvgl-codegen = { version = "0.6.2", path = "../lvgl-codegen" }
-lvgl-sys = { version = "0.6.2", path = "../lvgl-sys" }
+lvgl-sys = { version = "0.6.2", path = "../lvgl-sys", features = ["raw-bindings"]}
```

Comment:
v2 of the cargo feature resolver does not unify features on crates compiled multiple times. If this and the above both specify both features for `lvgl-sys` (instead of just 1 each), it should unify and so the double-warnings might disappear?

Reply:
That defeats the entire purpose of this PR, which is to prevent building the library during for lvgl's build.rs executable
The library will always be built twice, since one is compiled for xtensa (or riscv), the other for x86_64. So there is no way to unify that.

Reply:
Although this does only apply to cross-compilation, if `target == host` (no cross compile) it may compile it twice unnecessarily as well.

Reply:
Why not check for that in the build script & enable features from there by `println`ing `cargo:rustc-cfg=whatever`? Beyond `TARGET`, cargo also exports `HOST`, so a simple check together with the previous one might work (on this, maybe just check if those are the same instead? I wasn't able to find any docs on a `CROSS_COMPILE` variable)

Reply:
Can we gate this behind a flag or feature, then, so that we don't always compile twice? Feels like a pretty big cost still.

Reply:
Is this the only thing preventing this PR from landing?

If so, this can just have a `no_cross_compile` feature that enables both `library` and `raw-bindings` features of the `lvgl-sys` crate.

And this will apply to the normal dependency and the build dependency [confirmed](https://stackoverflow.com/questions/67897778/toggle-optional-build-dependencies-and-build-dependencies-using-features)